recipe/meta.yaml

```diff
@@ -23,26 +24,17 @@ requirements:
   host:
     - python
     - pip
-    - urllib3
-    # secure
-    - pyopenssl >=0.14
-    - cryptography >=1.3.4
-    - idna >=2.0.0
-    - certifi
-    - ipaddress  # [py<=33]
-    - pysocks >=1.5.6,<2.0,!=1.5.7
+    - setuptools
+    - wheel
   run:
     - python
-    - urllib3
-    # secure
-    - pyopenssl >=0.14
-    - cryptography >=1.3.4
-    - idna >=2.0.0
-    - certifi
-    - ipaddress  # [py<=33]
-    - pysocks >=1.5.6,<2.0,!=1.5.7
+    - urllib3 >=1.23
 
```

Comment:
It makes sense to add:
```
  run_constrained:
    - paramiko
```
see https://github.com/jelmer/dulwich/blob/d78de71a390e349106e7fb78e87255706dc75db9/setup.cfg#LL38C12-L38C20